homeassistant/components/steam_online/sensor.py

```diff
@@ -28,6 +28,9 @@
 STATE_LOOKING_TO_TRADE = "looking_to_trade"
 STATE_LOOKING_TO_PLAY = "looking_to_play"
 
+STEAM_API_URL = "https://steamcdn-a.akamaihd.net/steam/"
+STEAM_APP_ENDPOINT = "apps/"
```

Comment:
Combine these two constants into one:
```suggestion
STEAM_API_URL = "https://steamcdn-a.akamaihd.net/steam/apps/"
```